tests/test_part_image.py

```diff
@@ -316,7 +316,6 @@ def test_split_mender_conf(self, bitbake_variables, latest_part_image, bitbake_p
 class TestAllPartitionImages:
 
     @pytest.mark.min_yocto_version("warrior")
-    @pytest.mark.xfail(pytest.config.getoption('--test-conversion'), reason="Not yet working in mender-convert", raises=AssertionError, strict=True)
```

Comment:
@kacf I wonder if this `--test-conversion` flag will be unused now. Was it introduced only for this test?

Reply:
I guess it was. :-) Feel free to remove it.

Nice that it passes now!

Reply:
In short: we cannot remove the flag yet.

The flag is used in meta-mender fixtures to decide if we should expect Yocto environment or not. This fixtures files are actually copied from meta-mender when mender-convert executes tests. So, in order to clean this up, mender-convert should provide the non-Yocto variants of the required fixtures for mender-image-tests in its own repo, and then meta-mender could forget about this magic flag.

I don't think this is of any importance ATM so I'll let it be.